frontend/src/lib/candidate/components/textArea/TextArea.type.ts

```diff
@@ -6,7 +6,7 @@ export type TextAreaProps = {
   /**
    * The text content of the text area.
    */
-  text: string | undefined;
+  text?: string | undefined;
```

Comment:
If we add the `?`, we can then remove ` | undefined`, because they mean the same thing :D